### ISSUE TITLE
[Bug]: Duration in set_text doesn't seems to work

### ISSUE THREAD

**Issue by Remi123** [DmitriySalnikov/godot_debug_draw_3d#22]
### Godot version

4.1.1

### DebugDraw3D version

1.1.3

### On which operating systems the error occurs

Windows

### Using which renderers the error occurs

Vulkan

### Issue description

I'm using this addons for almost all my visual debugging. Good job.

The only issues I've found was that the duration in DebugDraw.set_text("A","B",0,Color(),2.5) doesn't leave the message displayed for 2.5. It show it but only very few frames.

### Steps to reproduce

From a fresh install

in any classes's ready function

```GDScript
func _ready():
  while(true):
    DebugDraw.set_text("A","B",0,Color(),2.5) # it's supposed to show A : B for 2.5 seconds
    await get_tree().create_timer(5.0,true,true).timeout # Wait 5 seconds
```

### Minimal reproduction project

N/A

**Comment by DmitriySalnikov** [comment]
Sorry for the long delay, I was doing other things and updating the OBSNotifier.

Regarding this problem, yes, I made a few logical mistakes. Now I will try to fix them.

---

![godot windows editor dev x86_64_V0L7EwH47A](https://github.com/DmitriySalnikov/godot_debug_draw_3d/assets/7782218/361678eb-aa24-4414-8e3a-ce79ffada448)

**Comment by DmitriySalnikov** [comment]
Fixed in 72d114b
https://github.com/DmitriySalnikov/godot_debug_draw_3d/actions/runs/6062785988

But there is one problem... I merged part of the update into the master branch ahead of time. Therefore, in the updated libraries, `DebugDraw` is divided into two separate classes: `DebugDraw2D` and `DebugDraw3D`.
